convert/chain.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+package convert
+
+import (
+	"fmt"
+	"reflect"
+
+	converter "github.com/anchore/go-struct-converter"
+
+	"github.com/spdx/tools-golang/spdx/common"
+	"github.com/spdx/tools-golang/spdx/v2/v2_1"
+	"github.com/spdx/tools-golang/spdx/v2/v2_2"
+	"github.com/spdx/tools-golang/spdx/v2/v2_3"
+)
+
+var DocumentChain = converter.NewChain(
+	v2_1.Document{},
+	v2_2.Document{},
+	v2_3.Document{},
+)
+
+// Document converts from one document to another document
```

Comment:
nit: add a couple words here to provide usage may be helpful since it may be tricky to figure out what the arguments look like without reading the code (especially that its expecting a pointer)